src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -36,7 +36,7 @@ public class NuGetPackageManager
     {
         private IReadOnlyList<SourceRepository> _globalPackageFolderRepositories;
 
-        private ISourceRepositoryProvider SourceRepositoryProvider { get; }
+        public ISourceRepositoryProvider SourceRepositoryProvider { get; }
```

Comment:
I guess we can revert this change or make it internal if we use `packageRestoreContext.SourceRepositories` as per https://github.com/NuGet/NuGet.Client/pull/4583/files#r859185269

Reply:
Reverted.